python/dask_cudf/dask_cudf/tests/test_accessor.py

```diff
@@ -527,3 +527,33 @@ def test_struct_explode(data):
     got = dgd.from_cudf(Series(data), 2).struct.explode()
     # Output index will not agree for >1 partitions
     assert_eq(expect, got.compute().reset_index(drop=True))
+
+
+def test_tz_localize():
+    data = date_range("2000-04-01", "2000-04-03", freq="H")
+    expect = Series(data).dt.tz_localize(
+        "US/Eastern", ambiguous="NaT", nonexistent="NaT"
+    )
+    got = dgd.from_cudf(Series(data), 2).dt.tz_localize(
+        "US/Eastern", ambiguous="NaT", nonexistent="NaT"
+    )
+    assert_eq(expect, got.compute())
```

Comment:
Unless there's a specific reason we need to here, I generally prefer passing the Dask object through uncomputed:

```suggestion
    assert_eq(expect, got)
```

So that we can get some additional checks run on the Dask divisions/graph in addition to the equality check

Reply:
Ah apologies didn't realize we were using cuDF's implementation of `assert_eq` here - maybe would make more sense to use Dask's:

```suggestion
    dd.assert_eq(expect, got)
```